clients/apps/web/src/hooks/queries/subscriptions.ts

```diff
@@ -1,15 +1,23 @@
-import { SubscriptionsApiListRequest } from '@polar-sh/sdk'
-import { useQuery } from '@tanstack/react-query'
+import { SubscriptionsApiListRequest, SubscriptionCancel } from '@polar-sh/sdk'
+import { useQuery, useMutation } from '@tanstack/react-query'
 
-import { api } from '@/utils/api'
+import { api, queryClient } from '@/utils/api'
 import { defaultRetry } from './retry'
 
+export const invalidateSubscriptionCache = (
+  organizationId: string,
+) => {
+  queryClient.invalidateQueries({
+    queryKey: ['subscriptions', organizationId],
```

Comment:
You can perfectly do:

```ts
queryClient.invalidateQueries({
    queryKey: ['subscriptions', { organizationId }],
})
```

Even if there are more keys in the object, Tanstack will match them (unless you specify `exact: true`). This allows to very finely invalidate queries.

Reply:
Awesome, will fix :)